apis/r/R/RcppExports.R

```diff
@@ -271,12 +283,18 @@ sr_set_dim_points <- function(sr, dim, points) {
     invisible(.Call(`_tiledbsoma_sr_set_dim_points`, sr, dim, points))
 }
 
+#' libtiledbsoma version
+#'
+#' Returns a string with version information for libtiledbsoma and the linked
+NULL
+
```

Comment:
What is this documenting?

Reply:
Fixed in #3017 -- sorry about this -- I should have caught it sooner. It's not part of this PR; the problem was on #3048. This PR simply  did a fresh roxygenise which surfaced the existing formatting error I had made.